examples/tracing_loops.py

```diff
@@ -70,4 +71,31 @@
 
 fig.tight_layout()
 
-plt.show()
+###############################################################################
+# Now, let's extract the intensities along the first detected loop and plot the intensity profile.
+
+# Convert the first loop to SkyCoord objects
+first_loop = np.array(loops[0])
+intensity_coords = trace_map.pixel_to_world(first_loop[:, 0] * u.pixel, first_loop[:, 1] * u.pixel)
+
+# Sample the intensity along the loop
+intensity = sunpy.map.sample_at_coords(trace_map, intensity_coords)
+
+# Calculate the angular separation along the loop
+angular_separation = intensity_coords.separation(intensity_coords[0]).to(u.arcsec)
```

Comment:
```suggestion
# Since we only currently get pixel locations, we need to get the word coordinates of the first loop.
first_loop = loops[0]
loop_coords = trace_map.pixel_to_world(first_loop[:, 0] * u.pixel, first_loop[:, 1] * u.pixel)

# Now we can extract the intensity along the loop
intensity = sunpy.map.sample_at_coords(trace_map, loop_coords)

# Finally, we can calculate the angular separation along the loop
angular_separation = intensity_coords.separation(loop_coords).to(u.arcsec)
```